collections/Tanner/MS_Tanner_15.xml

```diff
@@ -38,34 +37,67 @@
                   </altIdentifier>
                </msIdentifier>
                <msContents>
-                  <msItem class="#vitae_sanctorum" n="1" xml:id="MS_Tanner_15-item1">
-                     <title key="work_10467">Nova legenda Angliae</title>
-                     <textLang mainLang="la">Latin</textLang>
+                  <textLang mainLang="la">Latin</textLang>
+                  <msItem>
+                     <locus>(pp. v–viii)</locus>
+                     <title type="desc">Alphabetical summary</title>
+                     <incipit xml:lang="la">Adrianus abbas greca pingua peritus floruit.</incipit>
+                     <explicit xml:lang="la">Wulstanus baculum in marmore fixit. qui a nullo alio anelli potuit.</explicit>
+                  </msItem>
+                  <msItem>
+                     <locus>(pp. 1–581)</locus>
+                     <author key="person_22494546">John of Tynemouth</author>
+                     <title key="work_10467">Sanctilogium Angliae Walliae Scotiae et Hiberniae</title>
+                     <incipit xml:lang="la">Adrianus nacione affricanus comes et cooperator</incipit>
+                     <explicit xml:lang="la">sed ab epantu diectus plane dixit.</explicit>
+                     <colophon xml:lang="la">Perfectum est hoc opus wlgariter intitulatum De sanctis Anglie. Ad laudem et honorem omnipotentis dei. ac sancte Cantuar’ ecclesie. ex Impensis Reuerendi in cristo patris domini Thome Goleston eiusdem ecclesie Prioris. Ac sacrarum litterarum professoris egregii. Per me Jacobum Neell’ Normannum ac Rothomagi natum. Anno verbi incarnati .1.4.9.9. Deo gracias.</colophon>
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc form="codex">
                      <supportDesc material="perg">
                         <support>parchment</support>
+                        <extent>i (modern paper endleaf) + 4 (medieval endleaves and contents list, pp. i–viii) + 298 (pp. 1–596) + 4 (medieval endleaves) <dimensions type="leaf" unit="mm">
+                              <height>365</height>
+                              <width>270</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Early modern pagination. A nineteenth-century hand has annotated pp. 581–596 with further numbers, pp. 585–600, but the original pagination is accurate.</foliation>
```

Comment:
@holfordm I cannot see why this additional pagination was added: if you ever have a chance to glance at the original I wonder whether I am missing something. If they appear to have been added in error, I wonder whether we should cross out the added numbers to avoid confusing future readers.